internal/services/cdn/cdn_frontdoor_firewall_policy_resource.go

```diff
@@ -450,6 +450,57 @@ func resourceCdnFrontDoorFirewallPolicy() *pluginsdk.Resource {
 				},
 			},
 
+			"log_scrubbing": {
+				Type:     pluginsdk.TypeList,
+				MaxItems: 1,
+				Optional: true,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"enabled": {
+							Type:     pluginsdk.TypeBool,
+							Optional: true,
+							Default:  true,
+						},
+
+						"rule": {
+							Type:     pluginsdk.TypeList,
+							MaxItems: 100,
+							Optional: true,
+							Elem: &pluginsdk.Resource{
```

Comment:
Is there a default rule / behaviour here if `log_scrubbing` is enabled, but no rules are defined? Just wondering, if there's no default behaviour, if these should be `Required` / `MinItems` instead to ensure the block does something?

(The test cases suggest that your design is fine, but I wanted to be sure since it's a little confusing to have this without rules?)

```suggestion
						"enabled": {
							Type:     pluginsdk.TypeBool,
							Required: true,
						},

						"rule": {
							Type:     pluginsdk.TypeList,
							MaxItems: 100,
							Minitems: 1,
							Elem: &pluginsdk.Resource{
```

Reply:
Yes, there is a default behavior, if you just have the `log_scrubbing` enabled with no `rule`s defined it will automatically scrub what front door believes to be sensitive data. FWIW Portal acts the same way.